### ISSUE TITLE
When using Voiceover on Dashboard, unable to navigate to create new item.

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#4014]
When using Voiceover, I am unable to navigate to the create new item icon to select it.

<img width="69" height="90" alt="Image" src="https://github.com/user-attachments/assets/fd3c0298-89a7-4c5d-9a6c-a1f6596691d9" />
